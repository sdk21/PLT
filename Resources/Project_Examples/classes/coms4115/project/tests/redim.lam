matrix A = {
    1, 2, 3;
    4, 5, 6;
    7, 8, 9
};
dim A[2, 2];
print A;
dim A[3, 3];
print A;
