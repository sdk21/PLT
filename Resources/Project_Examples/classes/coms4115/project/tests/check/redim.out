1	2	
4	5	
1	2	0	
4	5	0	
0	0	0	
